bot/cogs/fun.py

```diff
@@ -199,6 +199,47 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @command(aliases=["leet"])
+    async def leetify(self, ctx: Context, *, content: str) -> None:
+        """Gives each letter of a given message a different markdown style."""
+
+        leetters = {
+            "a": ["A","4"], "b": ["b", "B"], "c": ["C", "c"], "d": ["D", "d"],
+            "e": ["E", "3"], "f": ["F", "f"], "g": ["G", "g"], "h": ["H", "h"],
+            "i": ["1", "I", "i"], "j": ["j", "J"], "k": ["K", "k"], "l": ["L", "l"],
+            "m": ["M", "m"], "n": ["n", "N"], "o": ["O", "0"], "p": ["P", "p"],
+            "q": ["Q", "q"], "r": ["R", "r"], "s": ["5", "S"], "t": ["t", "T"],
+            "u": ["U", "u"], "v": ["v","V"], "w": ["W", "w"], "x": ["X", "x"],
+            "y": ["y", "Y"], "z": ["Z", "z"]
+        }
+        content = [i.lower() + " " for i in ctx.message.content[1:]]
```

Comment:
Since you're getting `content` as an argument of that command now, there is no reason to use `ctx.message.content[1:]` it's exactly the same as doing `content` except `content` is more convenient.

```suggestion
        content = [i.lower() + " " for i in content]
```
also note that converting content to lowercase can be done in a much easier way like this:
```suggestion
        content = content.lower()
```
this can be done since `lower()` works for multi-letter strings too and does what you did automatically.

Reply:
You've committed the first suggested part, but as described below that, you don't actually need a list comprehension and convert each individual letter of the string into a list, you can simply use `.lower()` on the whole, string which will make all letters in it lowercase and use `list()` around it which will convert the string into a list of its words.

This is the preferred approach as `lower` is a built-in method running in `C` language instead of in python, which provides a faster way to convert string to lowercase instead of re-implementing it by that list comprehension